ember/app/components/project-detailed/component.js

```diff
@@ -28,4 +29,14 @@ export default class ProjectDetailedComponent extends Component {
     }
     this.store.findRecord('project', this.args.project.id);
   });
+
+  deleteProject = dropTask(async () => {
+    try {
+      this.args.project.destroyRecord();
+      this.router.transitionTo('projects.index');
+      this.notification.success('Project deleted successfully');
+    } catch (e) {
+      this.notification.danger(`An error occurred while deleting the project`);
+    }
+  });
```

Comment:
Let's make sure no one deletes a project by accident. You can use the `confirm` helper of `ember-uikit` for that.

Reply:
done here: https://github.com/adfinis/Outdated/pull/71/commits/85720bce36a97b047bb2ff050cba7842d284c780